src/report/core/report-schema-generator.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ReportSchemaGenerator } from './report-schema-generator';
+
+describe('ReportChangeDetector', () => {
+  const service = new ReportSchemaGenerator();
+
+  it('should detect fields in simple SELECT statement', () => {
+    const result = service.getTableNamesByQueries([
+      'SELECT c.name as Name, c.dateOfBirth as DateOfBirth, c.gender as Gender FROM Child c',
+    ]);
+
+    expect(result).toStrictEqual([['Name', 'DateOfBirth', 'Gender']]);
+  });
+
+  it('should detect fields in multiple simple SELECT statements', () => {
+    const result = service.getTableNamesByQueries([
+      'SELECT c.name as Name, c.dateOfBirth as DateOfBirth, c.gender as Gender FROM Child c',
```

Comment:
do we mandate an "as Name" alias in statements? Or is it okay to omit that?

Reply:
For now it easier this way... can be improved later.

Reply:
agreed, was just asking because that is something I need to follow when creating/updating actual ReportConfigs. If we don't change it in the next weeks, we should probably document this somewhere